addons/test_mail/tests/test_message_track.py

```diff
@@ -251,3 +252,30 @@ def cleanup():
             'description': 'Zizizatestmaildescription',
         })
         test_mail_record.with_context(default_parent_id=2147483647).write({'name': magic_code})
+
+    def test_track_invalid_selection(self):
```

Comment:
Try to order methods alphabetically in a given test class, as this is their execution order. That way test in console follow the same order as the file :)